src/markdown-pages/docs/cli-command-reference/cli-command-reference.md

```diff
@@ -28,10 +27,11 @@ Complete the following steps to start running the Galasa CLI tool:
 
 On Mac or Unix:
 
-1. Download the binary and re-name it to `galasactl`.
-2. Add `galasactl` to your PATH to enable the tool to be called from the command line without having to specify the path to the directory in which it is stored.
-3. Set execute permission on the binary by running the `chmod +x galasactl` command in the directory containing `galasactl`.
-4. Set permission to open the CLI tool by running the `spctl --add galasactl` command in the directory containing `galasactl`. You are prompted by a security panel asking you to log in to show that you are issuing the command.
+1. Find out the architecture of your machine by typing the command `uname -m` into your terminal. 
+2. Download the appropriate binary for your machine architecture and re-name it to `galasactl`.
+3. Add `galasactl` to your PATH to enable the tool to be called from the command line without having to specify the path to the directory in which it is stored.
```

Comment:
You could add "For example: `export PATH=${PATH}:/my/folder/containing/galasactl`